src/angular-vs-repeat.js

```diff
@@ -268,10 +268,17 @@
                         }
 
                         var rafReinitId;
+                        var isRafReinitAlreadyScheduled = false;
                         function rafReinit(){
                             if (reinitInRAF) {
-                                window.cancelAnimationFrame(rafReinitId);
-                                rafReinitId = window.requestAnimationFrame(reinitialize);            
+                                if (!isRafReinitAlreadyScheduled) {
+                                    isRafReinitAlreadyScheduled = true;
+                                    rafReinitId = window.requestAnimationFrame(function() {
```

Comment:
You can just re-use the rafReinitId instead of creating a new flag.

Reply:
Just set it to undefined when the callback executes.

Reply:
Thanks, but I prefer keeping the other variable for the sake of clarity.